grammars/csharp.tmLanguage.cson

```diff
@@ -2205,7 +2205,41 @@ repository:
             name: "variable.other.readwrite.cs"
       }
     ]
-  "declaration-expression":
+  "declaration-expression-local":
+    match: '''
+      (?x) # e.g. int x OR var x
+      (?:
+        \\b(var)\\b|
+        (?<type-name>
+          (?:
+              (?:(?<identifier>[_[:alpha:]][_[:alnum:]]*)\\s*\\:\\:\\s*)? # alias-qualification
+              (?<name-and-type-args> # identifier + type arguments (if any)
+                \\g<identifier>\\s*
+                (?<type-args>\\s*<(?:[^<>]|\\g<type-args>)+>\\s*)?
+              )
+              (?:\\s*\\.\\s*\\g<name-and-type-args>)* # Are there any more names being dotted into?
+              (?:\\s*\\*\\s*)* # pointer suffix?
+              (?:\\s*\\?\\s*)? # nullable suffix?
+              (?:\\s*\\[(?:\\s*,\\s*)*\\]\\s*)* # array suffix?
+          )|
+          (?<tuple>\\s*\\((?:[^\\(\\)]|\\g<tuple>)+\\))
+        )
+      )\\s+
+      \\b(\\g<identifier>)\\b\\s*
+      (?=[,)\\]])
+    '''
+    captures:
+      "1":
+        name: "keyword.other.var.cs"
+      "2":
+        patterns: [
+          {
+            include: "#type"
+          }
+        ]
+      "7":
+        name: "entity.name.variable.local.cs"
+  "declaration-expression-tuple":
```

Comment:
I'm not sure I follow why the name here is `declaration-expression-tuple`. Could you clarify?